backend/app/audio.py

```diff
@@ -1,28 +1,28 @@
 """audio processing, etc"""
 from pydub import AudioSegment
+import pandas as pd
+import numpy as np
+import soundfile as sf
```

Comment:
We generally keep these as their full length names. This makes the code throughout more readable as you don't need to return and check what `sf` refers to for example.